kornia/metrics/ssim.py

```diff
@@ -75,19 +75,19 @@ def ssim(
         raise ValueError(f"img1 and img2 shapes must be the same. Got: {img1.shape} and {img2.shape}")
 
     # prepare kernel
-    kernel: torch.Tensor = get_gaussian_kernel2d((window_size, window_size), (1.5, 1.5))
+    kernel: torch.Tensor = get_gaussian_kernel1d(window_size, 1.5)
```

Comment:
as suggested by @gau-nernst
```suggestion
    kernel: torch.Tensor = get_gaussian_kernel1d(window_size, 1.5, device=img1.device, dtype=img1.dtype)
```